cmd/kubeadm/app/cmd/upgrade/plan.go

```diff
@@ -50,25 +55,189 @@ func newCmdPlan(apf *applyPlanFlags) *cobra.Command {
 		applyPlanFlags: apf,
 	}
 
+	outputFlags := newUpgradePlanPrintFlags(output.TextOutput)
+
 	cmd := &cobra.Command{
 		Use:   "plan [version] [flags]",
 		Short: "Check which versions are available to upgrade to and validate whether your current cluster is upgradeable. To skip the internet check, pass in the optional [version] parameter",
 		RunE: func(_ *cobra.Command, args []string) error {
-			return runPlan(flags, args)
+			printer, err := outputFlags.ToPrinter()
+			if err != nil {
+				return err
```

Comment:
should we wrap this to have a more meaningful message?
not sure if the inner error already has this context.

```suggestion
				return errors.Wrap(err, "could not construct output printer")
```

also unclear if we are wrapping similarly for "kubeadm token ..."'s output.